src/shared/components/palette/datetime-picker/datetime-picker.component.ts

```diff
@@ -145,7 +127,7 @@ export class DatetimePickerComponent extends AbstractFormFieldComponent implemen
       this.hideTime = true;
       this.checkTime = false;
       this.dateTimeEntryFormat = this.formatTranslationService.removeTime(this.dateTimeEntryFormat);
-      this.momentFormat = 'YYYY-MM-DD';
+      this.momentFormat = 'YYYY-MM-DD'
```

Comment:
; is removed?

Reply:
@rayliangatos - this must've come in from the merge of `Release` into my PR branch... I'll fix that.

Reply:
@rayliangatos - this is now resolved.